src/main/java/com/bio4j/angulillos/TypedGraph.java

```diff
@@ -22,28 +22,26 @@
   default <
     VT extends VertexType<VT, G,RV,RE>
   >
-  VertexType<VT,G,RV,RE>.Vertex addVertex(VT vertexType) {
+  VT.Vertex addVertex(VT vertexType) {
 
-    return vertexType.fromRaw(
+    return vertexType.new Vertex(
```

Comment:
Actually, there is something funny happening. I've tried changing that many times and sometimes I cannot get rid of these errors, but then if I `clean` (_well_) and compile, it compiles fine. I just tried to make a little snippet of code with the same pattern and I couldn't reproduce this compilation error, so I think that it's some incremental compilation artefacts..